browser/src/Services/Explorer/ExplorerView.tsx

```diff
@@ -63,6 +66,31 @@ interface IMoveNode {
     }
 }
 
+const NodeTransitionWrapper = styled.div`
```

Comment:
Nice! Glad you were able to get this working 👍 Good example for how we can incorporate animations across our UI.